wp-includes/widgets/class-wp-widget-media-image.php

```diff
@@ -14,7 +14,7 @@
  *
  * @see WP_Widget
  */
-class WP_Widget_Image extends WP_Widget_Media {
+class WP_Widget_Media_Image extends WP_Widget_Media {
```

Comment:
Likewise here the docblock on line 3 is showing the old class name of `WP_Widget_Image`